docs/src/concepts.rst

```diff
@@ -3,4 +3,116 @@
 
 ======================
 Definition of concepts
-======================
\ No newline at end of file
+======================
+
+Variable Importance
+-------------------
+
+Global Variable Importance (VI) aims to assign a measure of
+relevance to each feature :math:`X^j` with respect to a target  :math:`y` in the
+data-generating process. In Machine Learning, it can be seen as a measure
+of how much a variable contributes to the predictive power of a model. We 
+can then define "important" variables as those whose absence degrades 
+the model's performance :footcite:p:`Covert2020`.
+
+So if ``VI`` is a variable importance method, ``X`` a variable matrix and ``y`` 
+the target variable, the importance of all the variables
+can be estimated as follows:
+
+.. code-block::
+
+    # instantiate the object
+    vi = VI()
+    # fit the models in the method
+    vi.fit(X, y)
+    # compute the importance and the pvalues
+    importance = vi.importance(X, y)
+    # get importance for each feature
+    importance = vi.importances_
+
+It allow us to rank the variables from more to less important.                            
+
+Here, ``VI`` can be a variable importance method implemented in HiDimStat,
+such as :class:`hidimstat.LOCO` (other methods will support the same API 
```

Comment:
If you can use the full name of the model before to introduce the acronym of it, it will be better.